reloj-checador/app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,65 +1,65 @@
-// body {
-//   background-color: #fff;
-//   color: #333;
-//   margin: 33px; }
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px; }
```

Comment:
try to use
```
body {
  background-color: #fff;
  color: #333;
  margin: 33px;
}
```
instead of
```
body {
  background-color: #fff;
  color: #333;
  margin: 33px; }
```
it's easier to read and maintain